canonical/snapcraft.io
# Migrate snaps page to react

## Done
- Added pagination to the snap list and registered snap list
- only the snaps' in that page data is shown in the chart
- migrated snaps page to react
- removed the status column

## How to QA
- go to https://snapcraft-io-4953.demos.haus/snaps
- verify the chart is rendered for the snaps on that page
- verify the snap list is rendered correctly

## Testing

## Issue / Card
Fixes [WD-17155](https://warthogs.atlassian.net/browse/WD-17155)



[WD-17155]: https://warthogs.atlassian.net/browse/WD-17155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
